snapshot/git/gitdb/README.md

```diff
@@ -13,6 +13,16 @@ GitDB is an implementation of ScootDB (snapshot.DB) that stores local values in
 ## Backends
 GitDB uses different Backends to identify, upload and download Snapshots.
 
+## Exit Codes
+GitDB defines a set of exit codes that are returned depending on various git related errors (see common/errors/exit_codes.go for definitive list):
+* Failed to git clean:        211
```

Comment:
I wouldn't even enumerate these here to avoid defining 2x, just direct to the above file

Reply:
👍